lib/screen/restaurant_menu_screen.dart

```diff
@@ -55,27 +58,69 @@ class RestaurantMenuScreen extends StatelessWidget {
               isBold: true,
               padding: const EdgeInsets.symmetric(vertical: 15),
             ),
-
             TextTitleSmall(
               text: restaurantDescription,
               padding: EdgeInsets.zero,
             ),
-
             const SectionHeaderTB(text: 'Uncategorized'),
             ..._getContent(),
-            const SizedBox(height: 16),
-            Align(
-              alignment: Alignment.center,
-              child: ElevatedButton(
-                onPressed: () {
-                  // Complete order functionality
-                },
-                child: const Text('Complete Order'),
-              ),
-            ),
+            const SizedBox(height: 90.0),
           ],
         ),
       ),
+      bottomSheet: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 8.0, horizontal: 15.0),
+        child: SizedBox(
+          height: 60.0,
+          child: Row(
+            children: [
+              Expanded(
+                  flex: 5,
+                  child: GestureDetector(
```

Comment:
Line 78 - 102 can be reduced using Button(text:...., Textcolor: ...., backgroundColor:...., onPressed:......)

Reply:
The shopping cart button is outlined button which has a border, I'm not sure if the current button.dart will suit this design.

Reply:
Since currently the button.dart only has option for background color, which fills the entire background, however, the shopping cart button has no background but only border.